packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -449,11 +456,11 @@ export default class Extension {
 
     assert(pair, 'Unable to find pair');
 
-    const remainingTime = this.refreshAccountPasswordCache(pair);
+    const { remainingTime } = await this.refreshAccountPasswordCache(pair);
 
     return {
-      isLocked: pair.isLocked,
-      remainingTime
+      remainingTime,
+      isLocked: pair.isLocked
```

Comment:
I thought we wanted to delete the `isLocked` property?

Reply:
If I remove it completely it was causing some tests and build to fail, so I thought keeping makes life a bit easier

Reply:
sure, then let's fix the value to `remainingTime > 0`, cause `pair.isLocked` is random

Reply:
You mean remainingTime <= 0, right?

Reply:
yes, right, sorry